src/_P221_AnalogOutPut.ino

```diff
@@ -0,0 +1,154 @@
+#include "_Plugin_Helper.h"
+#ifdef USES_P221
+//#######################################################################################################
+//#################################### Plugin 221: Analog OutPut ########################################
+//#######################################################################################################
+
+#include "src/Helpers/Rules_calculate.h"
+#include "src/WebServer/WebServer.h"
+
+#define PLUGIN_221
+#define PLUGIN_ID_221        221
+#define PLUGIN_NAME_221       "Analog OutPut - Out 0-3.3V "
+#define PLUGIN_VALUENAME1_221 "Output"
+
+boolean Plugin_221(byte function, struct EventStruct *event, String& string)
+{
+  boolean success = false;
+  static byte switchstate[TASKS_MAX];
+
+  switch (function)
+  {
+
+    case PLUGIN_DEVICE_ADD:
+      {
+        Device[++deviceCount].Number = PLUGIN_ID_221;
+        Device[deviceCount].Type = DEVICE_TYPE_SINGLE;
+        Device[deviceCount].VType = Sensor_VType::SENSOR_TYPE_SWITCH;
+        Device[deviceCount].Ports = 0;
+        Device[deviceCount].PullUpOption = false;
+        Device[deviceCount].InverseLogicOption = false;
+        Device[deviceCount].FormulaOption = false;
+        Device[deviceCount].ValueCount = 1;
+        Device[deviceCount].SendDataOption = true;
+        Device[deviceCount].TimerOption = false;
+        break;
+      }
+
+    case PLUGIN_GET_DEVICENAME:
+      {
+        string = F(PLUGIN_NAME_221);
+        break;
+      }
+
+    case PLUGIN_GET_DEVICEVALUENAMES:
+      {
+        strcpy_P(ExtraTaskSettings.TaskDeviceValueNames[0], PSTR(PLUGIN_VALUENAME1_221));
+        break;
+      }
+
+    case PLUGIN_GET_DEVICEGPIONAMES:
+      {
+        event->String1 = formatGpioName_output(F("Level low"));
+        break;
+      }
+
+    case PLUGIN_WEBFORM_LOAD:
+      {
+        // char tmpString[128];
+
+        addHtml(F("<TR><TD>Check Task:<TD>"));
```

Comment:
Shouldn't `addRowLabel(F("Check Task"))` better be used here? Easier maintenance might the UI layout be changed. (same below for "Check Value").